### PR TITLE
{tools}[GCCcore/10.2.0] coverage v5.5 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/coverage/coverage-5.5-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+version = '5.5'
+
+homepage = 'https://coverage.readthedocs.io'
+description = """ Coverage.py is a tool for measuring code coverage of Python programs.
+ It monitors your program, noting which parts of the code have been executed,
+ then analyzes the source to identify code that could have been executed but was not. """
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [PYPI_SOURCE]
```
Comment (Micket) [inline easybuild/easyconfigs/c/coverage/coverage-5.5-GCCcore-10.2.0.eb:13]: Please remove, this is the default for pythonpackages now

File: easybuild/easyconfigs/c/coverage/coverage-5.5-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+]
+
+use_pip = True
```
Comment (Micket) [inline easybuild/easyconfigs/c/coverage/coverage-5.5-GCCcore-10.2.0.eb:25]: Please add sanity_pip_check = True

**Review by Micket — DISMISSED** [review]

**Comment by SimonPinches** [comment]
@Micket Thanks for the comments.  I've tidied up the EB file as requested.

File: easybuild/easyconfigs/c/coverage/coverage-5.5-GCCcore-10.2.0.eb (hunk 2 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/c/coverage/coverage-5.5-GCCcore-10.2.0.eb:24]: Done :-)

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/eb253cfda026d629a452b345b4a4db3c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/aa9c70160a6b2e865a713dc7db64a834 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**